src/ORM/Query.php

```diff
@@ -793,6 +930,10 @@ public function __clone()
      */
     public function count()
     {
+        return $this->getCount();
+    }
+
+    public function getCount() {
```

Comment:
I too would stick with count(). The query builders are supposed to read like sql when possible.